components/function-controller/Gopkg.toml

```diff
@@ -12,17 +12,33 @@ required = [
 # controller-runtime, and dep fails due to Knative's requirements otherwise.
 
 [[override]]
-  name = "k8s.io/client-go"
-  version = "kubernetes-1.13.4"
+  name = "k8s.io/kubernetes"
+  version = "=v1.13.4"
 
 [[override]]
   name = "k8s.io/api"
   version = "kubernetes-1.13.4"
 
+[[override]]
+  name = "k8s.io/apiserver"
+  version = "kubernetes-1.13.4"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.13.4"
```

Comment:
Now we have Knative Serving 0.11 in our charts, does it make sense to upgrade kubernetes deps to 1.15.x